mkdocs.yml

```diff
@@ -16,6 +16,7 @@ nav:
     - Hyperopt: hyperopt.md
     - Edge Positioning: edge.md
     - Utility Subcommands: utils.md
+    - Exchanges: exchanges.md
```

Comment:
"Exchange-specific Notes: exchanges.md"
?

Reply:
I was thinking about this too and found it "too long" ...

Reply:
we'll see it in place...